lib/compress.js

```diff
@@ -69,6 +69,7 @@ function Compressor(options, false_by_default) {
         if_return     : !false_by_default,
         inline        : !false_by_default,
         join_vars     : !false_by_default,
+        keep_classnames: !false_by_default,
```

Comment:
I see what's going on - this should default to `false`.

Reply:
I bet that extra character is annoying you to no end. :-)

Reply:
Thanks for catching this - I waited till just before bedtime to make this PR to alleviate my pain.